panel/tests/pane/test_alert.py

```diff
@@ -36,7 +36,7 @@ def test_existing_css_classes():
     assert set(alert.css_classes) == {"alert", "alert-info", "important"}
 
 
-def test_all_view():
+def manualtest_all_view():
```

Comment:
Does it mean these tests will actually no longer run on the CI?

Reply:
Correct. The `manualtest` have no assert in them. It could, of course, find errors in the code, but mostly they are creating a servable dashboard. 

If any test with `return` in it also has an `assert`, I have made two tests one with the assert named `test_name` and one with return named `manualtest_name`.

Reply:
Oh ok I was unlucky then as this one has a couple of `assert`s. I didn't know there were kind of manuel tests in the test suite. Did you know? Just thinking that code that is not executed is code than can rot fast, if it's of no use maybe we could just get rid of it.

Reply:
I think we should either remove them or move them to a separate folder for now. So it is clear that these tests should be checked manually.